src/EventStore.Core/TransactionLog/Scavenging/Sqlite/SqliteScavengeBackend.cs

```diff
@@ -55,16 +62,19 @@ public class SqliteScavengeBackend<TStreamId> : SqliteScavengeBackend, IScavenge
 			var collisionStorage = new SqliteCollisionScavengeMap<TStreamId>();
 			CollisionStorage = collisionStorage;
 
-			var hashes = new SqliteScavengeMap<ulong, TStreamId>("HashUsers");
+			var hashes = new SqliteScavengeMap<ulong, TStreamId>("HashUsers", "INT");
```

Comment:
is it because the "HashUsers" map is backed by a cache that we're able to use an autoincrementing rowid key here to speed up accumulation? otherwise, queries done during accumulation would be slower.